### PR TITLE
Fine tuning audio translations

### PR DESCRIPTION
**Pull request by JoshCLWren**

### CODE REVIEW

File: audio_conversion.py [hunk 1]
```diff
@@ -7,23 +7,26 @@
...
         os.mkdir(folder_name)
-    output_file = f"/tmp/{podcast_title}.mp3"
+    output_file = f"/tmp/{title}.mp3"
     urllib.request.urlretrieve(url, output_file)
     download_time = time.time() - start_time
     print(f"Download time: {download_time}")
-    song = pydub.AudioSegment.from_mp3(output_file)
-
-    wav_file = f"{podcast_title}.wav"
+    if format == "mp3":
```
Comment (diemer) [inline audio_conversion.py]: The actual wav conversion doesn't start for 10 lines. That's a good clue that this method is violating the single responsibility principle.

Can you extract the file stuff above to it's own method, and wrap a call to that and this in a new method like `create_wav_file` ?

File: audio_conversion.py [hunk 2]
```diff
@@ -7,23 +7,26 @@
...
+        song = pydub.AudioSegment.from_mp3(output_file)
+    else:
+        song = pydub.AudioSegment.from_file(f"{output_file}", format=format)
```
Comment (diemer) [inline audio_conversion.py]: Can this or `from_mp3` ever fail? If so what happens? Do you need to guard against that?

File: audio_conversion.py [hunk 3]
```diff
@@ -7,23 +7,26 @@
...
+    wav_file = f"{title}.wav"
     song = song.set_channels(1)
     song = song.set_frame_rate(16000)
+    song = song.normalize()
     os.remove(output_file)
     end_time = time.time()
     total_time = end_time - start_time
```
Comment (diemer) [inline audio_conversion.py]: Total time seems unused.

File: audio_conversion.py [hunk 4]
```diff
@@ -7,23 +7,26 @@
...
-    wav_file = f"{podcast_title}.wav"
+    if format == "mp3":
+        song = pydub.AudioSegment.from_mp3(output_file)
+    else:
+        song = pydub.AudioSegment.from_file(f"{output_file}", format=format)
+    wav_file = f"{title}.wav"
     song = song.set_channels(1)
     song = song.set_frame_rate(16000)
+    song = song.normalize()
     os.remove(output_file)
```
Comment (diemer) [inline audio_conversion.py:28]: Seems like some clean up that is unrelated to wav conversion. Might be good to move to the suggested wrapper method.

File: audio_conversion.py [hunk 5]
```diff
@@ -32,32 +35,13 @@ def wav_converter(url, podcast_title):
...
 
 
-def video_to_wav(url):
+def video_to_audio(url):
     """
     Converts a youtube file to a mono wav file.
     """
-    start_time = time.time()
 
     stream_url = pytube.YouTube(url).streams.filter(only_audio=True).first().url
```
Comment (diemer) [inline audio_conversion.py:50]: Will first ever come back empty? If so will this blow up?

File: transcriber.py [hunk 6]
```diff
@@ -38,7 +37,7 @@ def get_large_audio_transcription(path, **episode):
         # adjust this per requirement
         silence_thresh=sound.dBFS - 14,
         # keep the silence for 1 second, adjustable as well
-        keep_silence=500,
+        keep_silence=True,
```
Comment (diemer) [inline transcriber.py]: Could be cool to move these hardcoded values to an ENV file

File: audio_conversion.py (hunk 3 above)
Comment (JoshCLWren) [reply audio_conversion.py]: I'm just printing it to the console. I'm not finding value in that though anymore so I think I'll just remove the timestamps here.

File: audio_conversion.py (hunk 2 above)
Comment (JoshCLWren) [reply audio_conversion.py]: good point, I'm not sure but I wrapped them in a try block

File: audio_conversion.py (hunk 1 above)
Comment (JoshCLWren) [reply audio_conversion.py]: love the feedback. I followed your advice.

File: audio_conversion.py (hunk 5 above)
Comment (JoshCLWren) [reply audio_conversion.py:50]: pytube could just stop working as it's want to do. I'll put an error handler around this.

File: transcriber.py (hunk 6 above)
Comment (JoshCLWren) [reply transcriber.py]: good thinking bro
